v8/payloads/channel.ts

```diff
@@ -207,7 +209,7 @@ export interface APIMessage {
 	 *
 	 * See https://discord.com/developers/docs/topics/permissions#role-object
 	 */
-	mention_roles: string[];
+	mention_roles: Pick<APIRole, 'id'>[];
```

Comment:
are these... objects? Because I think you meant `APIRole['id']`

Reply:
ah yeah oops